src/components/Login/Login.css

```diff
@@ -0,0 +1,4 @@
+.loginForm {
+    display: flex;
+    flex-direction: column;
+}
```

Comment:
You can target your class `.loginScreen` and make the login screen looking good. Start off with adjusting the width and add `margin: 0 auto`